test/e2e/app-dir/react-owner-stacks-svgr/react-owner-stacks-svgr.test.ts

```diff
@@ -0,0 +1,20 @@
+import { nextTestSetup } from 'e2e-utils'
+
+describe('react-owner-stacks-svgr', () => {
+  const { next, isNextStart, isTurbopack } = nextTestSetup({
+    files: __dirname,
+    packageJson: { dependencies: { '@svgr/webpack': '8.1.0' } },
+  })
+
+  /* eslint-disable jest/no-standalone-expect */
+  // Turbopack currently only supports `next dev` and does not support `next
+  // build`: https://nextjs.org/docs/architecture/turbopack#unsupported-features
```

Comment:
Not officially but we should start testing it. If it breaks in Turbopack build, it should be excluded via manifest instead so that we start tracking this test.

Reply:
I thought that with `skip` it would be added to the `pending` section in the manifest by the auto-generated PRs. Should it be in the `failed` section instead?

Reply:
Are we able to parse this test to know it's skipped? I assumed we need to declare that in the manifest and not at runtime.

Reply:
I think this should work. If I'm interpreting the scripts correctly, the info is parsed from `nextjs-test-results.json` that's part of the [`test-results-turbopack-production` artifact](https://github.com/vercel/next.js/actions/runs/9657309268/artifacts/1634797377). And the test status (e.g. `pending`) is included there.

Here's an example for a newly introduced skipped test: https://github.com/vercel/next.js/pull/64508/files#diff-ba07b6748eea52172a14f9255df869f843e6cfe179bf9a9b4472e9892d82e0e1R15. The manifest was then updated accordingly in https://github.com/vercel/next.js/pull/65055/files#diff-9031f7a1d624386e22ffc1d796e12e23337f4469777023d50bfee690158833d0R3464